javascript-custom-methods/calculator.js

```diff
@@ -0,0 +1,33 @@
+/* exported calculator */
+const calculator = {
+  add: function (x, y) {
+    return x + y;
+  },
+  subtract: function (x, y) {
+    return x - y;
+  },
+  multiply: function (x, y) {
+    return x * y;
+  },
+  divide: function (x, y) {
+    return x / y;
+  },
+  square: function (x) {
+    return x * x;
+  },
+  sumAll: function (numbers) {
+    var sum = 0;
+    for (var i = 0; i < numbers.length; i++) {
+      sum += numbers[i];
+    }
+    return sum;
+  },
+  getAverage: function (numbers) {
+    var sum = 0;
+    for (var i = 0; i < numbers.length; i++) {
+      sum += numbers[i];
+      var average = sum / numbers.length;
```

Comment:
Small suggestion — with this, average is being defined every time loop loops. What we can do is just return `sum / numbers.length` outside of loop when `sum` is all done calculating

Reply:
Got it!